<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Water</class>
 <widget class="QMainWindow" name="Water">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>852</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="waterstackedWidget">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>3</y>
      <width>400</width>
      <height>852</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>852</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>852</height>
     </size>
    </property>
    <widget class="QWidget" name="sex">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>395</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="sexbg">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>82</y>
         <width>526</width>
         <height>829</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/background/waterimg/sexbg.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nexttoweight">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>616</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 110, 156);
color: rgb(255, 255, 255);
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="female">
       <property name="geometry">
        <rect>
         <x>227</x>
         <y>288</y>
         <width>110</width>
         <height>110</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
border-radius: 50%;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/icons/waterimg/Female.png</normaloff>:/icons/waterimg/Female.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="male">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>288</y>
         <width>110</width>
         <height>110</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
border-radius: 50%;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/icons/waterimg/Male.png</normaloff>:/icons/waterimg/Male.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>78</x>
         <y>407</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(184, 184, 184);</string>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>407</y>
         <width>95</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(184, 184, 184);</string>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="backtomain">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>75</y>
         <width>46</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/buttons/waterimg/back.png</normaloff>:/buttons/waterimg/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="drink">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QWidget" name="drinkwaterwidget" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-107</y>
        <width>400</width>
        <height>964</height>
       </rect>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>828</y>
         <width>410</width>
         <height>900</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="waterButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>662</y>
        <width>114</width>
        <height>167</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="water.qrc">
        <normaloff>:/buttons/waterimg/drinkbutton.png</normaloff>:/buttons/waterimg/drinkbutton.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>60</y>
        <width>314</width>
        <height>418</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/waterimg/text.png);
background: none;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="percent">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>452</y>
        <width>58</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
background-color: transparent;
color: rgb(66, 174, 220);
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="recommend">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>198</y>
        <width>57</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
color: rgb(0, 110, 156);
background-color: transparent;
</string>
      </property>
      <property name="text">
       <string>1680</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ml">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>376</y>
        <width>120</width>
        <height>44</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>393</width>
        <height>110</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
background-color: transparent;
color: rgb(66, 174, 220);
</string>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="backtomain2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>46</width>
        <height>53</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="water.qrc">
        <normaloff>:/buttons/waterimg/back.png</normaloff>:/buttons/waterimg/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reco">
     <widget class="QWidget" name="recowidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>395</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="recobg">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>0</y>
         <width>393</width>
         <height>852</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/background/waterimg/recobg.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nextodrink">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>532</y>
         <width>161</width>
         <height>53</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="mlreco">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>427</y>
         <width>113</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(1, 102, 145);
border-radius: 20px;</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="backtoweather">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>55</y>
         <width>46</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/buttons/waterimg/back.png</normaloff>:/buttons/waterimg/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="weight">
     <widget class="QWidget" name="weightwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>395</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="weightbg">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>82</y>
         <width>677</width>
         <height>890</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/background/waterimg/weightbg.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nexttoweather">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>616</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 110, 156);
color: rgb(255, 255, 255);
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="kg">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>374</y>
         <width>113</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 110, 156);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="backtosex">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>75</y>
         <width>46</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/buttons/waterimg/back.png</normaloff>:/buttons/waterimg/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="weather">
     <widget class="QWidget" name="weatherwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>395</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="weatherbg">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>82</y>
         <width>526</width>
         <height>829</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/background/waterimg/weatherbg.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nexttoreco">
       <property name="geometry">
        <rect>
         <x>118</x>
         <y>616</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 110, 156);
color: rgb(255, 255, 255);
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="hot">
       <property name="geometry">
        <rect>
         <x>77</x>
         <y>305</y>
         <width>233</width>
         <height>62</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 110, 156);
border-radius: 20px;
text-align: left;</string>
       </property>
       <property name="text">
        <string>     Hot</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="temperate">
       <property name="geometry">
        <rect>
         <x>77</x>
         <y>380</y>
         <width>233</width>
         <height>62</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 110, 156);
background-color: rgb(235, 235, 235);
border-radius: 20px;
text-align: left;</string>
       </property>
       <property name="text">
        <string>     Temperate</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="cold">
       <property name="geometry">
        <rect>
         <x>77</x>
         <y>453</y>
         <width>233</width>
         <height>62</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 110, 156);
border-radius: 20px;
text-align: left;</string>
       </property>
       <property name="text">
        <string>     Cold</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>248</x>
         <y>315</y>
         <width>52</width>
         <height>197</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/icons/waterimg/weathericon.png);
background: none;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="backtoweight">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>75</y>
         <width>46</width>
         <height>53</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="water.qrc">
         <normaloff>:/buttons/waterimg/back.png</normaloff>:/buttons/waterimg/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="save">
     <widget class="QLabel" name="bg2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>393</width>
        <height>852</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/waterimg/waterbg.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="water.qrc"/>
 </resources>
 <connections/>
</ui>
